client-v2/src/components/FrontsEdit/ArticleFragmentFormInline.tsx

```diff
@@ -308,39 +310,49 @@ class FormComponent extends React.Component<Props, FormComponentState> {
       );
     };
 
+    const renderKickerSuggestion = (
```

Comment:
It would be nice to see this as a class method -- at the moment, this lambda is re-evaluated each render. The same applies to `getKickerContents` further down.